src/main/java/edu/iu/uits/lms/courselist/config/SecurityConfig.java

```diff
@@ -73,7 +73,7 @@ protected void configure(HttpSecurity http) throws Exception {
                     .withObjectPostProcessor(new LmsFilterSecurityInterceptorObjectPostProcessor())
                     .and()
                     .headers()
-                    .contentSecurityPolicy("style-src 'self'; form-action 'self'; frame-ancestors 'self' https://*.instructure.com")
```

Comment:
See if we can add another style-src so we can keep this but accommodate keeping this